libraries/monitor/src/monitor.js

```diff
@@ -245,7 +245,7 @@ class Monitor {
    *
    */
   reportError(err, level = 'err', extra = {}) {
-    if (err.hasOwnProperty && !(err.hasOwnProperty('stack') || err.hasOwnProperty('message'))) {
+    if (!(Object.prototype.hasOwnProperty.call(err, 'stack') || Object.prototype.hasOwnProperty.call(err, 'message'))) {
```

Comment:
Did you mean to make use of `in` instead of `hasOwnProperty` here? In any case, this looks good to me.

Reply:
This is investigating an "unknown" object that might be a class instance, so I stuck with `hasOwnProperty` to avoid capturing any prototype methods.  The other cases where I used `in` are looking at regular key/value "objects" where that's not likely to be an issue.